test/purchases_flutter_test.dart

```diff
@@ -5,6 +5,16 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:purchases_flutter/purchases_flutter.dart';
 
 // ignore_for_file: deprecated_member_use_from_same_package
+
+void _performDartSideChannelMethodCall(String methodName, dynamic params) {
```

Comment:
This is the only part that I'm not no familiar with so I don't want to blindly approve 😇

Reply:
Could you add a comment here explaining the need and usage of this?

Reply:
i.e.: 
```dart
/// testing utility that allows you to emit messages as if they were coming directly from the SDK
/// this is useful for ...
/// usage: ...
```

Reply:
oh this is cool, I need it for the setLogHandler